File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.py
class Solution:
    def totalMoney(self, n: int) -> int:
        numb=[1,2,3,4,5,6,7]
        day=int(n/7)
        return 28*(day)+7*(int(((day-1)*(day))/2))+(sum(numb[:n%7])+(n%7)*day)

File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.c
int somma(int n){
    int i;
    int numbers[]={1,2,3,4,5,6,7};
    int somma=0;
    for(i=0;i<n;i++){
        somma=somma+numbers[i];
    }
    return somma;
}

int totalMoney(int n){
    return (28*(n/7))+(7*(((n/7-1)*(n/7))/2))+somma(n%7)+((n%7)*(n/7));
}

